librz/util/x509.h

```diff
@@ -1,3 +1,7 @@
+// SPDX-FileCopyrightText: 2018 pancake <pancake@nopcode.org>
+// SPDX-FileCopyrightText: 2018 deroad <wargio@libero.it>
```

Comment:
i'm the owner of this, not pancake. same for all ASN1 related files.

Reply:
Right, sorry, files were renamed and git doesn't show history right for the renamed files.

Reply:
Fixed.